### ISSUE TITLE
use of xarray

### ISSUE THREAD

**Issue by aymeric-spiga** [aymeric-spiga/planetoplot#8]
planetoplot could use xarray, which would make a lot more versatile and less complicated library

**Comment by rpennel** [comment]
Indeed, xarray is quite powerful and does right out of the box lots of things we always try to put in a homemade library.

**Comment by aymeric-spiga** [comment]
https://github.com/pydata/xarray is a beast. I can do almost everything I was able to do with the homemade arguably DIY planetoplot, with all the additional power I have not explored yet, and all the professional coding and support and docs. Many thanks @rpennel for the comment! The only thing I needed in addition to xarray was simple CLI access (see https://github.com/pydata/xarray/issues/2034) but I made a short program that does the job https://github.com/aymeric-spiga/xarray-cli

**Comment by aymeric-spiga** [comment]
many thanks too to @jvatantollone who got me wondering about alternate solutions with his opendap question #9

**Comment by aymeric-spiga** [comment]
Adding geoviews in the mix could be useful http://geoviews.org/gallery/bokeh/xarray_image.html#bokeh-gallery-xarray-image
